.github/workflows/linting.yml

```
@@ -1,40 +1,22 @@
 name: Linting
 on:
+  push:
+    branches:
+      - master
   pull_request:
-    paths:
-      - '**/*.yaml'
-      - '**/*.yml'
   workflow_dispatch:
+
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
+
 jobs:
   lint-ruby:
-    name: Ruby - ${{ matrix.os }} ${{ matrix.ruby }}
-    runs-on: ${{ matrix.os }}-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu, windows]
-        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', 'head']
-        include:
-          - os: ubuntu
-            ruby: jruby-9.4
-          - os: windows
-            ruby: mingw
-        exclude:
-          - os: windows
-            ruby: head
     steps:
-      - name: Windows Specific
-        if: matrix.os == 'windows'
-        run: |
-          # Work around `Layout/EndOfLine: Carriage return character detected`
-          git config --system core.autocrlf false
       - uses: actions/checkout@v4
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
+          ruby-version: '3.3'
```

https://github.com/ruby/setup-ruby/blob/master/README.md#supported-version-syntax mentions that you can use
> - engine only like `ruby` and `truffleruby`, uses the latest stable release of that implementation
```suggestion
          ruby-version: ruby
```

---

Oh, thats nice. I was looking at docs but didn't find this